tests/unit/account_info_test.py

```
@@ -100,6 +149,8 @@ def test_from_proto(proto_account_info):
     assert account_info.token_relationships == []
     assert account_info.account_memo == "Test account memo"
     assert account_info.owned_nfts == 5
+    assert account_info.max_automatic_token_associations == 10
+    assert account_info.staking_info == None
```

Ruff (0.14.14)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)